library/core/src/ops/range.rs

```diff
@@ -369,7 +369,6 @@ impl<Idx> RangeInclusive<Idx> {
     #[lang = "range_inclusive_new"]
     #[stable(feature = "inclusive_range_methods", since = "1.27.0")]
     #[inline]
-    #[rustc_promotable]
```

Comment:
We need to keep allowing this one because it's used in the desugaring of `..=`:

```
[INFO] [stdout] 361 |             ast::ClassAsciiKind::Cntrl => &[0x00..=0x1f, 0x7f..=0x7f],
[INFO] [stdout]     |                                            ^^^^^^^^^^^^^^^^^^^^^^^^^-
[INFO] [stdout]     |                                            |                        |
[INFO] [stdout]     |                                            |                        temporary value is freed at the end of this statement
[INFO] [stdout]     |                                            creates a temporary which is freed while still in use
```